internal/verifier/verifier.go

```diff
@@ -122,3 +128,47 @@ func (verifier *AmberProvenanceMetadataVerifier) Verify(provenanceFilePath strin
 
 	return nil
 }
+
+// Internal intermediate representation of Provenances for verification.
```

Comment:
Golang comments start with the name of the element; in this case `ProvenanceIR`. The linters currently don't enforce it, but it is good to be consistent. Ref go/go-style/decisions#doc-comments.